commit d953a05d04b17b43f3bf4a183131e5c5e223ecaa
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 7 19:52:05 2010 -0700

    staging: brcm80211: Use linux/bitops.h instead of bitfuncs.h
    
    Use ffs() from linux/bitops.h, instead of the locally defined find_msbit().
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/phy/wlc_phy_lcn.c
+++ b/drivers/staging/brcm80211/phy/wlc_phy_lcn.c
@@ -21,7 +21,7 @@
 #include <linux/string.h>
 #include <linuxver.h>
 #include <siutils.h>
-#include <bitfuncs.h>
+#include <linux/bitops.h>
 #include <hndpmu.h>
 
 #include <wlc_phy_radio.h>
@@ -5196,7 +5196,7 @@ int32 wlc_lcnphy_rx_signal_power(phy_inf
 	{
 		uint32 power = (received_power * 16);
 		uint32 msb1, msb2, val1, val2, diff1, diff2;
-		msb1 = find_msbit(power);
+		msb1 = ffs(power) - 1;
 		msb2 = msb1 + 1;
 		val1 = 1 << msb1;
 		val2 = 1 << msb2;
